src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -13,7 +13,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <OutputPath>$(MicrosoftAndroidSdkOutDir)</OutputPath>
     <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
-    <DefineConstants>$(DefineConstants);TRACE;HAVE_CECIL;MSBUILD;ANDROID_24</DefineConstants>
```

Comment:
There is a block above like:
```csharp
#if ANDROID_25
		public string?    RoundIcon       {get; set;}
#endif
```
Is it possible this doesn't actually work? Do we need to define all of them?

Reply:
There are tests for it, so I assume it works:
https://github.com/xamarin/xamarin-android/blob/789773ec9989e665000ee47793a3910b82891f5f/src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/ManifestTest.cs#L936

I think it uses some sort of alternative method, as the value is processed in a local variable since the property isn't available:
https://github.com/xamarin/xamarin-android/blob/789773ec9989e665000ee47793a3910b82891f5f/src/Xamarin.Android.Build.Tasks/Mono.Android/IntentFilterAttribute.Partial.cs#L16